test/signs/realtime_test.exs

```diff
@@ -540,15 +540,15 @@ defmodule Signs.RealtimeTest do
       Signs.Realtime.handle_info(:run_loop, @sign)
     end
 
-    test "does not allow ARR on second line if platform does not have multiple berths" do
+    test "allows ARR on second line" do
```

Comment:
Is there value in keeping this as a distinct test case? I suspect if we were writing this from scratch and this "multi-berth" feature had never existed, we wouldn't think to have it, since it's not covering any unique code path. (Though maybe we'd want to test that ordering works correctly when the displayed time is "ARR" for both entries.)

Reply:
I thought about that too. Obviously it's nice to see the delta to confirm that this change works, but there may not be a ton of value beyond that. I'm inclined to leave it for now, but it could be a candidate for sweeping up down the road, certainly.